### DISCUSSION TITLE
Domain [domain] was added to a different project. Please complete verification to add it to this project instead.

### DISCUSSION THREAD

**Discussion by lockieluke** [vercel/community#5144] [Help]
### Summary

Domain: lockie.dev

I accidentally transferred my domain to dub.co without realising, now that I've deleted my project on dub.co, the domain is still not released

![image](https://github.com/vercel/community/assets/25424409/91c31abc-facc-46ed-8264-ad6c85886dcc)

### Steps to Reproduce

1. Transfer your domain to dub.co
2. Delete the dub.co project
3. There's no way of unlinking the domain from dub.co, they're not releasing it

**Comment by BitBeaver-182** [comment]
I have the same problem. Did you manage to solve the issue?

**Comment by lockieluke** [comment]
I contacted Dub via email and they managed to release my domain

**Comment by lockieluke** [comment]
And also go to vercel, on the root dashboard not in your project, go to domains and delete the dub related keys in the dns table

**Comment by guvenkaranfil** [comment]
I have the same problem. I've created a project and add my custom domain without verifying it then I remove the project and created a fresh one but when i add my custom domain the error appeared
